integration/package.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+if [[ -z "${FPC_PATH}" ]]; then
+  echo "Error: FPC_PATH not set"
+  exit 1
+fi
+
+packageChaincode() {
+  output_dir=$1
+  cc_name=$2
+  cc_version=$3
+  cc_type=$4
+  port=$5
+
+  tmp_dir=$(mktemp -d -t "${cc_name}-package")
+  mkdir -p "${output_dir}"
+
+  for p in "${PEERS[@]}"; do
+    local out_dir=$tmp_dir/$p
+
+    mkdir -p $out_dir
+    addr="$cc_name.$p:$port"
+    createConnection $out_dir $addr
+    echo $out_dir
+    ls $out_dir
+    cat $out_dir/connection.json
+  done
+
+  tar cfz $tmp_dir/code.tar.gz --exclude=code.tar.gz -C $tmp_dir .
+
+  output="${output_dir}/${cc_name}.tgz"
+
+  createMetafile $tmp_dir $cc_name $cc_version $cc_type
+
+  tar cfz "${output}" -C "$tmp_dir" code.tar.gz metadata.json
+
+  rm -rf "$tmp_dir"
+}
+
+createMetafile() {
+  out_dir=$1
+  cc_name=$2
+  cc_version=$3
+  cc_type=$4
+
+  #{
+  #  "type": "external",
+  #  "label": "ercc_1.0"
+  #}
+
+  jq -n '$ARGS.named' \
+    --arg type "${cc_type}" \
+    --arg label "${cc_name}_${cc_version}" \
+    >>"$out_dir/metadata.json"
+}
+
+createConnection() {
+  out_dir=$1
+  address=$2
+  timeout=10s
+  use_tls=false
+
+  #{
+  #  "address": "ercc.org1.example.com:9999",
+  #  "dial_timeout": "10s",
+  #  "tls_required": false
+  #}
+
+  jq -n '$ARGS.named' \
+    --arg address "${address}" \
+    --arg dial_timeout "${timeout}" \
+    --argjson tls_required "${use_tls}" \
+    >>"$out_dir/connection.json"
+}
+
+DEPLOYMENT_PATH=$(PWD)/_deployment
```

Comment:
Same problem as at other place: did that really work for you that way?
```suggestion
DEPLOYMENT_PATH=${PWD}/_deployment
```

Reply:
oh on a mac this works; inside the docker-dev container, it doesn't. good catch